default.config.yml

```diff
@@ -215,6 +216,7 @@ extra_packages:
 extra_security_enabled: false
 
 drush_version: "master"
+drush_path: "{{ drupal_composer_install_dir }}/vendor/drush/drush/drush"
```

Comment:
I'd rather add the task to determine the path and use default if it's not installed via Drupal VM.